index.js

```diff
@@ -61,17 +71,41 @@ module.exports = {
             }
         }, options);
 
+        /** Plugins */
+        eleventyConfig.addPlugin(EleventyRenderPlugin);
+        eleventyConfig.addPlugin(pluginWebc);
+
         /** Filters */
         eleventyConfig.addFilter("formatDate", formatDateFilter);
         eleventyConfig.addFilter("isoDate", isoDateFilter);
         eleventyConfig.addFilter("limit", limitFilter);
-        eleventyConfig.addFilter("markdown", markdownFilter);
+        eleventyConfig.addFilter("markdown", function (value) {
+            // eslint-disable-next-line no-console
+            console.warn("This filter will be removed in a future version of eleventy-plugin-fluid. Use the renderString shortcode instead.");
```

Comment:
Probably worth adding the name of the filter in the message. Or does that already get logged along with this?